software_tests

```diff
@@ -1,13 +1,10 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
 #
 # various tests that gnssrefl build is still working
 # this only checks that the code does not crash. It does not currently
-rm -rf /Users/kristine/testing_friday
-mkdir /Users/kristine/testing_friday
-## use new environment variables
-export ORBITS=/Users/kristine/testing_friday
-export REFL_CODE=/Users/kristine/testing_friday
-
-# do not overwrite EXE for now
```

Comment:
Got rid of this since when running within the Docker image the necessary environment variables are already set.